src/coreclr/vm/comutilnative.h

```diff
@@ -121,6 +121,7 @@ typedef GCMemoryInfoData * GCMEMORYINFODATA;
 typedef GCMemoryInfoData * GCMEMORYINFODATAREF;
 #endif // USE_CHECKED_OBJECTREFS
 
+typedef void (*EnumerateConfigurationValuesCallback)(int type, int64_t intValue, BOOL boolValue);
```

Comment:
```suggestion
using EnumerateConfigurationValuesCallback = void (*)(int type, int64_t intValue, BOOL boolValue);
```

@jkoritzinsky and I are trying to introduce regular C++ style into the codebase.

Reply:
I think you want `bool` here, not `BOOL`.